data/domain/world-6/summoning.tsx

```diff
@@ -99,41 +141,74 @@ export class SummonBonus {
     pristineCharmBonus: number = 1;
     artifactBonus: number = 0;
     taskBoardBonus: number = 0;
+    achievement379Bonus: number = 0;
+    achievement373Bonus: number = 0;
+    summoning32Bonus: number = 0;
 
     constructor(public index: number, public data: SummonEnemyBonusModel, bonusValue: number = 0) {
         this.bonusValue = bonusValue;
     }
 
     getBonus = (): number => {
-        return 3.5 * this.bonusValue * this.pristineCharmBonus * 
-            (1 + ((this.artifactBonus + this.taskBoardBonus) / 100));
+        switch (true)
+        {
+            // +1 from in-game indexes as it starts from 1 instead of 0
```

Comment:
I missed this in first pass, are you saying we have a bug in the data (repo) that the indexes don't match in-game indexes? 🤔

Reply:
Well, it's Lava being inconsistent himself, the bonus indexes are starting from 1 in-game too, but for some reason when he calls the function to retrieve the value from those bonuses it uses a 0-based index, so depending on which part of the game code you read you need to be careful which one is used...

Reply:
Sigh .. it is what it is haha.